app/component/SummaryPage.js

```diff
@@ -2501,6 +2513,7 @@ class SummaryPage extends React.Component {
                   focusToPoint={this.focusToPoint}
                   focusToLeg={this.focusToLeg}
                   isMobile={false}
+                  carItinerary={carPlan?.itineraries}
```

Comment:
Maybe here we should just fetch the first itinerary (there will always be just one or none), so the carItinerary can be singular.

Reply:
Changed to send only the first one on.